.ci/common.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# Attention, there is no "-x" to avoid problems on Travis
+set -e
+
+# result of command goes to $RUN_JOB
+# 0 = not run (skip)
+# 1 = run (no skip)
+function should_run_job {
+    SKIP_JOB_BY_FILES=$1
```

Comment:
please use `local` keyword for defining local variables, to avoid unexpected side effects.

Reply:
Done.